apps/meteor/ee/client/sidebar/footer/SidebarFooterWatermark.tsx

```diff
@@ -1,16 +1,19 @@
-import { Box } from '@rocket.chat/fuselage';
+import { Box, Skeleton } from '@rocket.chat/fuselage';
 import { useTranslation } from '@rocket.chat/ui-contexts';
```

Comment:
change the `useTranslation` hook  to the `useTranslation` from the `react-18next`

Reply:
actually both are pretty much the same so in theory there ir no need to change